trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @decoration.correct_name.upcase[0...10]
```

Comment:
- Hey, the `upcase` will make the names in uppercase. As you are meant to capitalize the name, this will capitalize the first letter. e.g 'Jitender Kumar!'.upcase = #=> JITENDER KUMAR!, 'Jitender Kumar!'.capitalize = #=> Jintender kumar! 
Kindly fix this issue.
- Also you want to return 10 characters, [0...10] will return 11 characters, you may want to use [0...9] which will return 10 characters.